src/components/TodoContent/TodoContent.tsx

```diff
@@ -0,0 +1,139 @@
+import { FC, useEffect, useState } from 'react';
+
+import { TodoHeader } from '../Header/TodoHeader';
+import { TodoMain } from '../Main/TodoMain';
+import { TodoFooter } from '../Footer/TodoFooter';
+
+import { Todo } from '../../types/types';
+
+export const TodoContent: FC = () => {
+  // #region State
+  const [todos, setTodos] = useState<Todo[]>(() => {
+    const localValue = localStorage.getItem('todos');
+
+    return localValue ? JSON.parse(localValue) : [];
+  });
+
+  const [filteredTodos, setFilteredTodos] = useState(todos);
```

Comment:
it's redundant to create the separate state for filteredTodos, we can just create the var

```
const filteredTodos = getFilteredPeople(people)
```
and in that function you will filter the todos to get the filteredTodos array